src/dev/i18n/serializers/json.test.js

```diff
@@ -32,6 +32,6 @@ describe('dev/i18n/serializers/json', () => {
       ],
     ]);
 
-    expect(serializeToJson(messages)).toMatchSnapshot();
+    expect(serializeToJson([...messages])).toMatchSnapshot();
```

Comment:
__nit:__ well, it doesn't make sense to use `Map` in this test if we always convert it to the array of entries, so maybe let's just get rid of `Map` altogether here.